viewing_party/main.py

```diff
@@ -0,0 +1,129 @@
+# Wave 1 s
+def create_movie(movie_title,genre,rating):
+    movie = {}
+    if movie_title == None or genre == None or rating == None:
+        return None
+    else: 
+        movie["title"] = movie_title
+        movie["genre"] = genre
+        movie["rating"] = rating
+    print(movie)
```

Comment:
oops don't forget to get rid of any comments or print statements that are for debugging